web_monitoring/db.py

```diff
@@ -434,18 +434,18 @@ def diff_version(version_uuid, versions, diffs, source_type,
                  source_metadata)
 
 
-class WebVersioningException(Exception):
+#class WebVersioningException(Exception):
```

Comment:
Is this change intentional? At least some of this are used in the module, so I believe commenting them out will break things.